File: python/6kyu/Stop_gninnipS_My_sdroW.py
def spin_words(sentence):
    sList = sentence.split(" ")
    result = ([])
    #print(list(sList[0]).reverse())
    
    for si in sList:
        if len(si) > 4:
            st = list(si)
            st.reverse()
            si = "".join(st)
        result.append(si)
        
    # Your code goes here
    return " ".join(result)

File: csharp/6kyu/Stop_gninnipS_My_sdroW.cs
using System.Collections.Generic;
using System.Linq;
using System;

public class Kata
{
  public static string SpinWords(string sentence)
  {
    string [] sent = sentence.Split(" ");
    List<string> result = new List<string>();
    char[] cArray;
    
    foreach(var i in sent){
      if ( i.Length >= 5){
        cArray = i.ToCharArray();
        Array.Reverse(cArray);
        result.Add( new string ( cArray ) );
      }else result.Add(i);
            
    }
    return string.Join(" ", result);
    
    
  }
}